src/Unleash/Internal/CachedFilesLoader.cs

```diff
@@ -1,113 +1,220 @@
+using System;
 using System.IO;
-using Unleash.Events;
+using System.Linq;
 using Unleash.Logging;
-using Unleash.Scheduling;
 
 namespace Unleash.Internal
 {
-    internal class CachedFilesLoader
+    public interface BackupManager
```

Comment:
Nitpick, but we usually name interfaces prepended with an I (IBackupManager)

Reply:
Cheers, I forgot about this madness in .NET. I'll patch it